packages/sourcecred/src/api/bundledPlugins.js

```diff
@@ -1,23 +1,36 @@
 // @flow
 
 import type {Plugin} from "./plugin";
+import {type PluginId, getPluginOwner} from "./pluginId";
 import {GithubPlugin} from "../plugins/github/plugin";
 import {DiscoursePlugin} from "../plugins/discourse/plugin";
 import {DiscordPlugin} from "../plugins/discord/plugin";
 import {InitiativesPlugin} from "../plugins/initiatives/plugin";
 import {EthereumPlugin} from "../plugins/ethereum/plugin";
+import {
+  ExternalPlugin,
+  ExternalPluginIdOwner,
+} from "../plugins/external/plugin";
+import {DiskStorage} from "../core/storage/disk";
 
 /**
  * Returns an object mapping owner-name pairs to CLI plugin
  * declarations; keys are like `sourcecred/github`.
  */
 // TODO(@decentralion): Fix the type signature here.
-export function bundledPlugins(): {[pluginId: string]: Plugin} {
-  return {
+export function getPlugin(pluginId: PluginId): ?Plugin {
+  const mapping = {
     "sourcecred/github": new GithubPlugin(),
     "sourcecred/discourse": new DiscoursePlugin(),
     "sourcecred/discord": new DiscordPlugin(),
     "sourcecred/initiatives": new InitiativesPlugin(),
     "sourcecred/ethereum": new EthereumPlugin(),
   };
+  if (mapping[pluginId.toString()]) return mapping[pluginId.toString()];
+  if (getPluginOwner(pluginId) === ExternalPluginIdOwner)
+    return new ExternalPlugin({
+      pluginId,
+      storage: new DiskStorage(process.cwd()),
+    });
+  return null;
```

Comment:
just curious: why not an implicit `undefined` return here?

Reply:
will change, probably just a years-old habit from the 1 year in college I spent writing java lol.